app/services/invoices/customer_usage_service.rb

```diff
@@ -98,7 +98,7 @@ def boundaries
 
     def compute_amounts
```

Comment:
Would be nice to call `Invoices::ComputeAmountsFromFees` no?

Reply:
Fees are not persisted in this class, so sadly it's not working